PDF/PDFWindow.xaml.cs

```diff
@@ -342,6 +344,28 @@ private static TreeViewItem VisualUpwardSearch(DependencyObject source)
       IPDFViewer.ExtractBookmark(bookmark);
     }
 
+    private async void TvBookmark_MenuItem_PDFExtractWithPriority(object sender,
+                                            RoutedEventArgs e)
+    {
+
+      var result = await Forge.Forms.Show.Window()
+        .For(new Prompt<double> { Title = "Bookmark Priority?", Value = -1 });
+
+      if (!result.Model.Confirmed)
+      {
+        return;
+      }
+
+      PdfBookmark bookmark = (PdfBookmark)tvBookmarks.SelectedItem;
+      if (bookmark == null)
+      {
+        return;
+      }
+
+      IPDFViewer.ExtractBookmark(bookmark, result.Model.Value);
```

Comment:
Can you add a check for the Priority value provided by the user, like in your other Extract with priority PR ?

Reply:
OK. I added a check and included an Alert window if the priority value is invalid.